src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -115,8 +119,9 @@ export class TerminalQuickFixAddon extends Disposable implements ITerminalAddon,
 		decoration?.onRender((e: HTMLElement) => {
 			if (!this._decorationMarkerIds.has(decoration.marker.id)) {
 				this._currentQuickFixElement = e;
-				e.classList.add(DecorationSelector.QuickFix, DecorationSelector.Codicon, DecorationSelector.CommandDecoration, DecorationSelector.XtermDecoration);
-				e.style.color = '#ffcc00';
+				e.classList.add(DecorationSelector.QuickFix, DecorationSelector.LightBulb, DecorationSelector.Codicon, DecorationSelector.CommandDecoration, DecorationSelector.XtermDecoration);
+				// must be inlined
+				e.style.color = '#ddb100';
```

Comment:
This should live in css so we can pull the theme's color, otherwise we'll get accessibility bugs. Should be able to use the `!important` trick here too